PSScripts/Test-BranchFeatures.ps1

```diff
@@ -0,0 +1,37 @@
+<#
+.SYNOPSIS
+Tests the current branch for build features
+
+.DESCRIPTION
+Determine which build features that a build could run.
+
+.EXAMPLE
+Test-BuildFeatures.ps1
+
+#>
+
+[System.Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSProvideCommentHelp', '', Scope='Function', Target='Test-IfNotFeatureBranch')]
+[System.Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSProvideCommentDescription', '', Scope='Function', Target='Test-IfNotFeatureBranch')]
+
+param()
```

Comment:
2 param blocks

Reply:
Once is for the script-level param (which is required for the SuppressMessage attributes),  one is for the function itself.